### PR TITLE
Fix dependency issues (rebased).

### PR DESCRIPTION
**Pull request by ewlsh**
This fixes Dockerfile issues (with compilation) and updates the dependencies with security and bug fixes. This is rebased on the last pull request. WebKit is removed to allow the system to compile/run correctly but will be added back in a later pull.

### CODE REVIEW

File: lib/tasks/gir.thor [hunk 1]
```diff
@@ -21,19 +21,25 @@ class GirCLI < Thor
     end
     glob.each do |path|
       puts 'Generating scraper for ' + File.basename(path) + '...'
+      begin
```
Comment (ptomato) [inline lib/tasks/gir.thor]: Indentation is a bit weird in this block ... is it somehow mixing tabs and spaces?

File: lib/tasks/gir.thor [hunk 2]
```diff
@@ -21,19 +21,25 @@ class GirCLI < Thor
...
   desc 'generate', 'Generate a scraper from a GIR file'
   def generate(gir_path)
-    gir = read_gir gir_path
-
-    namespace = gir.root.elements['namespace']
-    scraper_info = process_namespace namespace
-    scraper_info[:slug] = generate_slug scraper_info
-    scraper_info[:version] = compute_version gir, scraper_info
-    write_scraper gir_path, scraper_info
+
```
Comment (ptomato) [inline lib/tasks/gir.thor]: This block doesn't seem to have been modified except for whitespace, so it'd be best to leave it as it was.

File: Dockerfile [hunk 3]
```diff
@@ -22,14 +22,14 @@ RUN dnf update -y && \
         libnotify-devel libpeas-devel librsvg2-devel libsecret-devel \
         libzapojit-devel pango-devel polkit-devel poppler-glib-devel \
         rest-devel telepathy-glib-devel tracker-devel udisks-devel \
-        upower-devel vte{,291,3}-devel webkitgtk{,3,4}-devel
```
Comment (ptomato) [inline Dockerfile:25]: No webkit docs anymore?

File: lib/docs/scrapers/gnome/gir_scraper.rb [hunk 4]
```diff
@@ -34,7 +34,9 @@ def doctool
 
     def run_doctool(gir_path)
       puts 'Generating HTML documentation...'
-      dir = Dir.mktmpdir
+      dir = Dir.mktmpdir 
+
+      puts "cmd: #{doctool} #{gir_path} -o #{dir} -l gjs -f devdocs"
```
Comment (ptomato) [inline lib/docs/scrapers/gnome/gir_scraper.rb]: Was this intentional or was it left over from debugging?

**Review by ptomato — CHANGES_REQUESTED** [review]
In general looks good, just a few comments mostly around formatting. Thanks!

File: Dockerfile (hunk 3 above)
Comment (ewlsh) [reply Dockerfile:25]: Discussed (removed do to issues with compilation/breakage, will be added later)

**Review by ptomato — APPROVED** [review]
OK, looks good now, thanks!

**Pull request merged**